src/System.Private.ServiceModel/tools/test/Bridge.Build.Tasks.targets

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask AssemblyFile="$(WcfToolsOutputPath)Bridge.Build.Tasks.dll" TaskName="ReleaseBridgeResourcesTask" />
+
+  <!-- ReleaseBridgeResources:
+      This target asks the Bridge (if running) to release its resources.
+      This is necessary so that the content of the resource folder can change.
+  -->
+  <Target Name="ReleaseBridgeResources">
+    <ReleaseBridgeResourcesTask
+      BridgeHost="$(BridgeHost)"
```

Comment:
The Bridge design isn't able to handle multiple clients (other than from the same client machine using the same set of resources).  At the moment, it is a kind of remote singleton.  We're going to have to design a way to maintain a set of resources per IP client address or something like that.